packages/analyzer/src/model/warning.ts

```diff
@@ -13,7 +13,7 @@
  */
 
 import * as chalk from 'chalk';
-import stable from 'stable';
+import * as stable from 'stable';
```

Comment:
This file isn't a legal ES6 module, which is why I was using the `import stable = require('stable');` form before.

Reply:
Here's the actual fix: https://github.com/Polymer/tools/commit/f8daf698d306140448c1a63439b80a2b2d14f672

Reply:
I am not going to try and monkey with this again.  Sorry!

Reply:
(new ConeOfShame()).apply(usergenic);